files/build_templates/init_cfg.json.j2

```diff
@@ -27,6 +27,11 @@
             "POLL_INTERVAL": "10000"
         }
     },
+    "BGP_DEVICE_GLOBAL": {
+        "STATE": {
+	   "tsa_enabled": "false"
```

Comment:
>	    [](http://example.com/codeflow?start=0&length=4)

Please reformat json indentation. Do not mix spaces with tabs.

Reply:
thanks for the pointer, fixed the issue